src/cf-expandables/usage.md

```diff
@@ -164,15 +169,16 @@ Allows you to float information left and right.
 ## Recommended expandable pattern
 
 Expandables can be built by combining the basic barebones structure described
-in the next section along with more specialized expandable elements
-and modifiers described throughout.
+in the next section along with more specialized expandable elements and
+modifiers described throughout.
 
 ### Default state
 
 The following combination is our recommended go-to expandable pattern.
 
 <div class="o-expandable o-expandable__padded">
-    <button class="o-expandable_header o-expandable_target" title="Expand content">
+    <button class="o-expandable_header o-expandable_target"
+            title="Expand content">
```

Comment:
There was no specific reason for keeping the `title` attributes - I just tended to default to keeping whatever was there and just modifying what I had to!